packages/sdk-ts/src/core/authz/msgs/MsgExec.ts

```diff
@@ -0,0 +1,91 @@
+import { MsgExec as BaseMsgExec } from '@injectivelabs/chain-api/cosmos/authz/v1beta1/tx_pb'
+import { Message } from 'google-protobuf'
+import { Any } from 'google-protobuf/google/protobuf/any_pb'
+
+import { MsgBase } from '../../MsgBase'
+
+export declare namespace MsgExec {
+  export interface Params {
+    grantee: string
+    msgs: MsgBase<unknown, any, Message, unknown, unknown> | MsgBase<unknown, any, Message, unknown, unknown>[]
+  }
+
+  export interface DirectSign {
+    type: '/cosmos.authz.v1beta1.MsgExec'
+    message: BaseMsgExec
+  }
+
+  export interface Data extends BaseMsgExec.AsObject {
+    '@type': '/cosmos.authz.v1beta1.MsgExec'
+  }
+
+  export interface Amino extends BaseMsgExec.AsObject {
+    '@type': '/cosmos.authz.v1beta1.MsgExec'
+  }
+
+  export type Proto = BaseMsgExec
+}
+
+export default class MsgExec extends MsgBase<
+  MsgExec.Params,
+  MsgExec.Data,
+  MsgExec.Proto,
+  MsgExec.Amino,
+  MsgExec.DirectSign
+> {
+  static fromJSON(params: MsgExec.Params): MsgExec {
+    return new MsgExec(params)
+  }
+
+  toProto(): MsgExec.Proto {
+    const { params } = this
+
+    const message = new BaseMsgExec()
+    message.setGrantee(params.grantee)
+
+    if (Array.isArray(params.msgs)) {
```

Comment:
Here, instead of having these `if` conditions we can do 

```ts
const msgs = Array.isArray(params.msgs) ? msgs : [msgs]

params.msgs.forEach(msg => {
    const msgValue = new Any()
    msgValue.setTypeUrl(msg.toData()['@type'])
    msgValue.setValue(msg.toProto().serializeBinary())

    message.addMsgs(msgValue)
  });
  ```